app/presenters/dttp/trainee_presenter.rb

```diff
@@ -42,6 +42,7 @@ def programme_details_params(contact_change_set_id)
       {
         "dfe_ContactId@odata.bind" => "$#{contact_change_set_id}",
         "dfe_programmestartdate" => programme_start_date.in_time_zone.iso8601,
+        "dfe_programmeenddate" => programme_end_date.in_time_zone.iso8601,
```

Comment:
Can we check that this is the way we need / want to send the programme end date to DTTP? There's nothing on the trello card about sending to DTTP.

Reply:
All good

Reply:
I can confirm that this is indeed the format.

Reply:
👍